deploy/pubsub-pub.yaml

```diff
@@ -19,6 +19,9 @@ spec:
           - name: grpc
             containerPort: 80
         args: [
-            "-url=pubsub.atlas.svc:80",
+            "-url=pubsub.$PUBSUB_NAMESPACE.svc:80",
```

Comment:
Should we switch to a different default port for ClusterIP service? 80 seems like a strange choice.

Reply:
what's strange about port 80?

Reply:
port 80 is generally used for web client to receive HTML page. perhaps this makes it unconventional choice. But not an issue.